calibr8/utils.py

```diff
@@ -207,20 +207,17 @@ def plot_t_band(ax, independent, mu, scale, df, *, residual_type: typing.Optiona
 
 
 def plot_continuous_band(ax, independent, model, residual_type: typing.Optional[str]=None):
-    """Helper function for plotting the 68, 90 and 95 % likelihood-bands of a t-distribution.
+    """Helper function for plotting the 84, 95 and 97.5 % likelihood-bands of a univariate distribution.
```

Comment:
The numbers here are now different compared to the other functions.
I'm pretty sure the `68, 90 and 95` is correct, because the list in the code referes to the upper bound only.

Reply:
```suggestion
    """Helper function for plotting the 68, 90 and 95 % likelihood-bands of a univariate distribution.
```

Reply:
Are you sure you reviewed the latest version? I don't see that